lacommunaute/notification/utils.py

```diff
@@ -38,5 +40,20 @@ def collect_following_replies():
     ]
 
 
+def collect_topics_digest_for_tags():
+    last_notif = last_notification(kind=EmailSentTrackKind.TAG_DIGEST)
+
+    return [
+        (
+            f"{reverse('forum_conversation_extension:topics')}?tags={tag.slug}",
+            f"La communauté - résumé hebdo de {tag.name}",
+            TagsNotification.objects.filter(tags=tag.id, newpost=True).values_list("user__email", flat=True),
+            Topic.objects.filter(tags=tag, updated__gte=last_notif).count(),
+        )
+        for tag in TaggedItem.tags_for(TagsNotification)
+        if Topic.objects.filter(tags=tag, updated__gte=last_notif).count() > 0
+    ]
```

Comment:
Pour éviter de faire deux fois le compte en SQL.

Je pense qu’il est possible de tout calculer en une seule requête, mais je n’ose pas écrire la requête sans tests :see_no_evil: .

```suggestion
def collect_topics_digest_for_tags():
    last_notif = last_notification(kind=EmailSentTrackKind.TAG_DIGEST)

    digests = []
    for tag in TaggedItem.tags_for(TagsNotification):
        topics_count = Topic.objects.filter(tags=tag, updated__gte=last_notif).count()
        if topics_count > 0:
            digests.append(
                (
                    f"{reverse('forum_conversation_extension:topics')}?tags={tag.slug}",
                    f"La communauté - résumé hebdo de {tag.name}",
                    TagsNotification.objects.filter(tags=tag.id, newpost=True).values_list("user__email", flat=True),
                    topics_count,
                )
            )
    return digests
```